src/app/models/submission.server.model.ts

```diff
@@ -182,13 +182,21 @@ const EncryptSubmissionSchema = new Schema<
 EncryptSubmissionSchema.methods.getWebhookView = function (
   this: IEncryptedSubmissionSchema,
 ): WebhookView {
+  const attachmentRecords: Record<string, string> = {}
+  if (this.attachmentMetadata) {
+    for (const [k, v] of this.attachmentMetadata) {
+      attachmentRecords[k] = v
+    }
+  }
```

Comment:
nit: can be converted from Map to Object with

```ts
const attachmentRecords = Object.fromEntries(this.attachmentMetadata)
```

Reply:
^ just beware that this will cause a TypeError if `this.attachmentMetadata` is `undefined`, which it may be for submissions created before we had storage mode attachments. so we need to either retain the `if (this.attachmentMetadata.)` or do `Object.fromEntries(this.attachmentMetadata ?? new Map())`.